packages/terra-base/tests/nightwatch/NoTranslationsLoadedBase.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import Base from '../../lib/Base';
+
+const BaseNoTranslationsLoaded = () => {
+  const noTranslations = (
+    <Base locale="gabcdef" translationsLoadingPlaceholder={<div>No Translations</div>}>
```

Comment:
If it doesn't throw error message in phantomjs, I'm fine with keeping it. If it does throw error message in phantomjs, I vote to remove this nightwatch test.

The concern is, the nightwatch test will depend on I18n component and Base component. That makes it quite fragile. If we move or update the supported locales validation, it will probably fail the nightwatch test in here.